kitchen.yml

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+# For help on this file's format, see https://kitchen.ci/
+driver:
+  name: docker
+  use_sudo: false
+  privileged: true
+  run_command: /lib/systemd/systemd
+
+# Make sure the platforms listed below match up with
+# the `env.matrix` instances defined in `.travis.yml`
+platforms:
+  - name: debian-9-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:debian-9
+  - name: ubuntu-1804-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:ubuntu-18.04
+
+provisioner:
+  name: salt_solo
+  log_level: info
+  salt_install: none
+  require_chef: false
+  formula: apt
+  salt_copy_filter:
+    - .kitchen
+    - .git
+  pillars:
+    top.sls:
+      base:
+        '*':
+          - apt
+
+verifier:
+  # https://www.inspec.io/
+  name: inspec
+  sudo: true
+  # cli, documentation, html, progress, json, json-min, json-rspec, junit
+  reporter:
+    - cli
+
+suites:
+  - name: repositories
+    provisioner:
+      state_top:
+        base:
+          '*':
+            - apt.repositories
+            - apt.update
+      pillars_from_files:
+        apt.sls: test/integration/repositories/pillar.sls
```

Comment:
In terms of these `pillars_from_files` paths (including the one below), goes back to these discussions we had on Slack with @daks:

* https://freenode.logbot.info/saltstack-formulas/20190425#c2137081
* https://freenode.logbot.info/saltstack-formulas/20190425#c2137133 -- you involved in the discussion from here
* Conclusion in `postgres-formula` [PR](https://github.com/saltstack-formulas/postgres-formula/pull/262/files#diff-fdbca9ed07a634829a3548f874dc925aR97)

No way am I suggesting that we should continue with that.  But it would be nice to work towards a standard that can be recommended across formulas.

Reply:
the idea is to use `test/salt/{pillar, states}/` directories to store salt data used for tests

Reply:
Fixed